Running SLURM prolog script on pink55
===============================================================================
Job started on Sat Aug  4 18:57:52 BST 2018
Job ID          : 123696
Job name        : saliency-batch
WorkDir         : /mainfs/lyceum/grm1g17/lasagne4bio/test_saliency
Partition       : lyceum
Num hosts       : 1
Num cores       : 14
Num of tasks    : 14
Hosts allocated : pink55
Job Output Follows ...
===============================================================================
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
INFO (theano.gof.compilelock): Waiting for existing lock by process '28709' (I am process '50198')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
INFO (theano.gof.compilelock): Waiting for existing lock by unknown process (I am process '51275')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
INFO (theano.gof.compilelock): Waiting for existing lock by unknown process (I am process '51821')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
INFO (theano.gof.compilelock): Waiting for existing lock by process '53051' (I am process '52745')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
INFO (theano.gof.compilelock): Waiting for existing lock by process '29227' (I am process '52745')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
Traceback (most recent call last):
  File "saliency.py", line 14, in <module>
    from utils import decode, ssConvertMap, ssConvertString, convertPredictQ8Result2HumanReadable, window
ImportError: cannot import name 'window'
==============================================================================
Running epilogue script on pink55.

Submit time  : 2018-08-04T18:57:47
Start time   : 2018-08-04T18:57:52
End time     : 2018-08-04T19:03:01
Elapsed time : 00:05:09 (Timelimit=05:40:00)

Job ID: 123696
Array Job ID: 123693_3
Cluster: i5
User/Group: grm1g17/fp
State: FAILED (exit code 1)
Nodes: 1
Cores per node: 14
CPU Utilized: 00:01:13
**** Please note that the CPU efficency of your job is less than 50% ****
**** Ask the HPC team for advice -- if you need it
**** CPU Efficiency: 1.69% of 01:12:06 core-walltime

Memory Utilized: 1.10 GB

